src/apps/answers/tests/test_answers.py

```diff
@@ -4693,3 +4619,278 @@ async def test_include_ehr_parameter_in_data_endpoint(
         )
 
         assert _answer["ehrDataFile"] == filename
+
+
+@pytest.mark.usefixtures("mock_kiq_report")
+class TestCompletions(BaseTest):
```

Comment:
For your specific test cases, some of them rely on existing data, rather than creating test-specific data. This can lead to tests being a bit more fragile. Can we update that?